mobile/src/main/java/org/openhab/habdroid/ui/AbstractBaseActivity.kt

```diff
@@ -105,9 +109,8 @@ abstract class AbstractBaseActivity : AppCompatActivity(), CoroutineScope {
         val km = getSystemService(KEYGUARD_SERVICE) as KeyguardManager
         val locked = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) km.isDeviceSecure else km.isKeyguardSecure
         if (locked) {
```

Comment:
`else` show a Snackbar with "No screen lock set" and a button that takes the user to the Android settings? This Snackbar should be shown as well if the user sets up the screen lock in the openHAB settings.

Reply:
This seems a bit excessive to me, at least the proposed else branch. How about disabling the screen lock option in the settings activity if Android lock is disabled?

Reply:
If the pref is going to be disabled, the summary should be something like "No Android screen lock oder fingerprint set".

Reply:
Yeah, was thinking about something among those lines ... maybe a bit more verbose: `You need to enable a screen lock for your device for using this feature.`